core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1709,6 +1715,11 @@ get_bypass_media(JObj) ->
         'false' -> 'undefined'
     end.
 
+-spec get_rtcp_mux(kz_json:object()) -> boolean().
+get_rtcp_mux(JObj) ->
+    Default = kapps_config:get_is_true(?CONFIG_CAT, <<"default_rtcp_mux">>, 'true'),
```

Comment:
@lazedo is required to add default value to `core/kazoo_endpoint/src/kz_endpoint.hrl` file?